src/main/resources/i18n/Scalars.properties

```diff
@@ -10,20 +10,20 @@
 # REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
 # so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
 #
-Scalar.unexpectedAstType=Expected an AST type of ''{0}'' but it was a ''{1}''.
+Scalar.unexpectedAstType=Expected an AST type of ''{0}'' but it was a ''{1}''
```

Comment:
Nit: all the default validation messages did not end in a trailing full stop, so removing for consistency